src/main/scala/scalaz/zio/saga/Saga.scala

```diff
@@ -37,13 +37,52 @@ final case class Saga[+E, +A] private (
     })
 
   /**
-   * Flattens the structure of this saga by executing outer saga first and then executes inner saga.
+   * Flattens the structure of this Saga by executing outer Saga first and then executes inner Saga.
    * */
   def flatten[E1 >: E, B](implicit ev: A <:< Saga[E1, B]): Saga[E1, B] =
     self.flatMap(r => ev(r))
 
   /**
-   * Materializes this saga to ZIO effect.
+   * Returns Saga that will execute this Saga in parallel with other, combining the result in a tuple.
+   * Both compensating actions would be executed in case of failure.
+   * */
+  def zipPar[E1 >: E, B](that: Saga[E1, B]): Saga[E1, (A, B)] =
+    zipWithPar(that)((a, b) => (a, b))
+
+  /**
+   * Returns Saga that will execute this Saga in parallel with other, combining the result with specified function `f`.
+   * Both compensating actions would be executed in case of failure.
+   * */
+  def zipWithPar[E1 >: E, B, C](that: Saga[E1, B])(f: (A, B) => C): Saga[E1, C] = {
+    def coordinate[A1, B1, C1](f: (A1, B1) => C1)(
+      fasterSaga: Exit[(E1, Compensator[Any, E1]), (A1, Compensator[Any, E1])],
```

Comment:
type aliasing?

Reply:
I'll address this concerns in subsequent PR's